packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts

```diff
@@ -245,8 +245,8 @@ export const getStyles = (props: IPanelStyleProps): IPanelStyles => {
         width: panelWidth.full,
         selectors: {
           [HighContrastSelector]: {
-            borderLeft: `3px solid ${palette.neutralLight}`,
-            borderRight: `3px solid ${palette.neutralLight}`
+            borderLeft: `3px solid ${semanticColors.variantBorder}`,
```

Comment:
This is the first time I've done a color conversion, and most of the colors seemed pretty obvious except for this one and the footer border later. Not sure who to ask to confirm I used the right color here.

Reply:
`neutralLight` doesn't seem right for high contrast, Im expecting something like `borderRightColor: 'Highlight'` 
I'll ask @betrue-final-final for his thoughts!

Reply:
Yeah I thought that was weird too--I chose `variantBorder` for the new color since it's the same as `neutralLight` usually (and might make sense semantically?) but for sure open to other suggestions.

Reply:
Yeah, I see why you chose it! Also, Ben says the system color reference should be `'WindowText'`